arrows/colmap/CMakeLists.txt

```diff
@@ -0,0 +1,42 @@
+# TODO Make everything configurable with variables so developers dont have to reconfigure everytime
+set(plugin_colmap_headers
+    colmap.h
+)
+
+kwiver_install_headers(
+    SUBDIR arrows/colmap
+    ${plugin_colmap_headers}
+)
+
+kwiver_install_headers(
+    ${CMAKE_CURRENT_BINARY_DIR}/kwiver_algo_colmap_export.h
+    NOPATH SUBDIR arrows/colmap
+)
+
+set(
+    plugin_colmap_sources
+    colmap.cxx
+)
+
+kwiver_add_library(
+    kwiver_algo_colmap
```

Comment:
I am now getting several errors at link time

```
undefined reference to `boost::system::generic_category()'
```

This may be due to issues with multiple copies of boost on my system. Does everything compile and link for you?

Reply:
Yes, this was an issue I had before. I actually addressed these issues earlier but I must have wiped my changes that made this arrow use the package variables and wrapped the including of the arrow in an if statement